mm2src/coins/eth.rs

```diff
@@ -167,6 +168,7 @@ lazy_static! {
     pub static ref ERC1155_CONTRACT: Contract = Contract::load(ERC1155_ABI.as_bytes()).unwrap();
 }
 
+type EthPrivKeyPolicy = PrivKeyPolicy<KeyPair>;
```

Comment:
It could be better to place this type definition 3 lines below  to group it with other private types

Reply:
Done